Fix Travis Build

Travis currently shows the below error.

`ERROR in ./src/styles/main.scss
Module build failed: Error: Can't resolve '~@angular/material/prebuilt-themes/indigo-pink.css' in '/home/travis/build/fossasia/loklak_search/src/styles'
    at onError (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:61:15)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at /home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:40:4
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:144:11)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:249:35)
    at resolver.doResolve.createInnerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:44:6)
 @ multi ./src/styles/main.scss
ERROR in ./src/styles/main.scss
Module build failed: ModuleBuildError: Module build failed: Error: Can't resolve '~@angular/material/prebuilt-themes/indigo-pink.css' in '/home/travis/build/fossasia/loklak_search/src/styles'
    at onError (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:61:15)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at /home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:40:4
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:144:11)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:249:35)
    at resolver.doResolve.createInnerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:44:6)
    at runLoaders (/home/travis/build/fossasia/loklak_search/node_modules/webpack/lib/NormalModule.js:195:19)9m
    at /home/travis/build/fossasia/loklak_search/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /home/travis/build/fossasia/loklak_search/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/home/travis/build/fossasia/loklak_search/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Promise.resolve.then.then.catch (/home/travis/build/fossasia/loklak_search/node_modules/postcss-loader/lib/index.js:189:71)
    at process._tickCallback (internal/process/next_tick.js:109:7)
 @ ./src/styles/main.scss
 @ multi ./src/styles/main.scss
ERROR in ./src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in '/home/travis/build/fossasia/loklak_search/src'
resolve './$$_gendir/app/app.module.ngfactory' in '/home/travis/build/fossasia/loklak_search/src'
  using description file: /home/travis/build/fossasia/loklak_search/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /home/travis/build/fossasia/loklak_search/package.json (relative path: ./src)
    using description file: /home/travis/build/fossasia/loklak_search/package.json (relative path: ./src/$$_gendir/app/app.module.ngfactory)[0m
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory.ts doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory.js doesn't exist
      as directory
        /home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory doesn't exist
[/home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory]
[/home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory.ts]
[/home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory.js]
[/home/travis/build/fossasia/loklak_search/src/$$_gendir/app/app.module.ngfactory]
 @ ./src/main.ts 5:0-74
 @ multi ./src/main.ts
ERROR in /home/travis/build/fossasia/loklak_search/src/app/home/home.component.scss:712m
throw new Error("Module build failed: Error: Can't resolve '~@angular/material/prebuilt-themes/indigo-pink.css' in '/home/travis/build/fossasia/loklak_search/src/app/home'\n    at onError (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:61:15)\n    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)\n    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)\n    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)\n    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)\n    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)\n    at /home/travis/build/fossasia/loklak_search/nod
Error: Module build failed: Error: Can't resolve '~@angular/material/prebuilt-themes/indigo-pink.css' in '/home/travis/build/fossasia/loklak_search/src/app/home'
    at onError (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:61:15)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
[31m    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at /home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:40:4
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:252:11)
    at innerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/Resolver.js:144:11)
    at loggingCallbackWrapper (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:249:35)
    at resolver.doResolve.createInnerCallback (/home/travis/build/fossasia/loklak_search/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:44:6)
    at Object.Y/Sa (/home/travis/build/fossasia/loklak_search/src/app/home/home.component.scss:71:7)
    at __webpack_require__ (/home/travis/build/fossasia/loklak_search/src/app/home/home.component.scss:20:30)
    at Y/Sa (/home/travis/build/fossasia/loklak_search/src/app/home/home.component.scss:63:18)
    at /home/travis/build/fossasia/loklak_search/src/app/home/home.component.scss:66:10
    at ContextifyScript.Script.runInContext (vm.js:35:29)
    at ContextifyScript.Script.runInNewContext (vm.js:41:15)
    at Object.runInNewContext (vm.js:93:38)
    at WebpackResourceLoader._evaluate (/home/travis/build/fossasia/loklak_search/node_modules/@ngtools/webpack/src/resource_loader.js:110:40)
    at Compilation.compilation.plugin (/home/travis/build/fossasia/loklak_search/node_modules/@ngtools/webpack/src/resource_loader.js:58:26)
    at Compilation.applyPluginsAsyncSeries (/home/travis/build/fossasia/loklak_search/node_modules/tapable/lib/Tapable.js:206:13)
The command "ng build --prod --aot --build-optimizer" failed and exited with 1 during .
`

--- a/package.json
+++ b/package.json
@@ -44,10 +44,11 @@
     "sw-toolbox": "^3.6.0",
     "ts-helpers": "^1.1.1",
     "wicg-inert": "^1.1.4",
-    "zone.js": "^0.8.14"
+    "zone.js": "^0.8.14",
+    "enhanced-resolve":"3.3.0"
   },
   "devDependencies": {
-    "@angular/cli": "^1.5.0",
+    "@angular/cli": "^1.6.4",
     "@angular/compiler-cli": "4.3.5",
     "@angular/language-service": "4.3.5",
     "@types/jasmine": "~2.5.53",

